Jenkinsfile.update

```diff
@@ -0,0 +1,211 @@
+
+import org.kie.jenkins.MavenCommand
+import org.kie.jenkins.MavenStagingHelper
+
+deployProperties = [:]
+optaplannerRepository = 'optaplanner'
+vehicleRoutingRepository = 'optaweb-vehicle-routing'
+employeeRosteringRepository = 'optaweb-employee-rostering'
+kogitoExamplesRepository = 'kogito-examples'
+kogitoAppsRepository = 'kogito-apps'
+
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem16g'
+    }
+    tools {
+        maven 'kie-maven-3.6.2'
+        jdk 'kie-jdk11'
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '10', numToKeepStr: '')
+        timeout(time: 120, unit: 'MINUTES')
+    }
+    environment {
+        MAVEN_OPTS = '-Xms1024m -Xmx4g'
+
+        // creates HAS for PR branchname
+        BOT_BRANCH_HASH = "${util.generateHash(10)}"
+    }
+    parameters {
+        // GIT-INFORMATION
+        string(name: 'BUILD_BRANCH_NAME', defaultValue: 'master', description: 'Which branch to build? Set if you are not on a multibranch pipeline.')
+        string(name: 'GIT_AUTHOR', defaultValue: 'kiegroup', description: 'Which Git author repository ?')
+
+        // RELEASE INFORMATION
+        booleanParam(name: 'RELEASE', defaultValue: true, description: 'Is this build for a release?')
+
+        // VERSION UPDATE INFORMATION
+        string(name: 'NEW_SNAPSHOT_VERSION', defaultValue: '', description: 'Setup a specific build display name')
+        string(name: 'MAVEN_DEPENDENCIES_REPOSITORY', defaultValue: '', description: 'Maven repository where to find dependencies if those are not in the default JBoss repository.')
+        string(name: 'MAVEN_SETTINGS_CONFIG_FILE_ID', defaultValue: 'kogito_release_settings', description: 'Maven settings configfile to use in pipeline for Maven commands')
+        string(name: 'KOGITO_VERSION', defaultValue: '', description: 'Optional if not RELEASE. If RELEASE, cannot be empty.')
+
+        // PULLREQUEST INFORMATION - Bot author information. Set as params for easy testing.
+        string(name: 'GIT_AUTHOR_BOT', defaultValue: 'bsig-gh-bot', description: 'From which author should the PR be created ?')
+        string(name: 'BOT_CREDENTIALS_ID', defaultValue: 'bsig-gh-bot', description: 'Credentials for PR creation')
+    }
+
+    stages {
+        stage('Clean workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Checkout repositories') {
+            steps {
+                checkoutRepo(optaplannerRepository, getBuildBranch())
+                checkoutRepo(vehicleRoutingRepository, getBuildBranch())
+                checkoutRepo(employeeRosteringRepository, getBuildBranch())
+                checkoutRepo(kogitoExamplesRepository, getBuildBranch())
+                checkoutRepo(kogitoAppsRepository, getBuildBranch())
+            }
+        }
+        stage('Prepare for PR') {
+            when {
+                expression { return isRelease() }
```

Comment:
The job should be triggered only when we want to upgrade the versions, thus we don't need any conditional stages.